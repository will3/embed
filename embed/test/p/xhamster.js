const test = require('../utils/test');

test('xhamster',
  'https://xhamster.com/movies/7668106/sloppy_deepthroat.html');
