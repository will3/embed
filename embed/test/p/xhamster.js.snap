// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` xhamster 1`] = `
Object {
  "originalUrl": "https://xhamster.com/movies/7668106/sloppy_deepthroat.html",
  "site_name": "xHamster",
  "title": "Sloppy Deepthroat: Free Amateur HD Porn Video db - xHamster",
  "video": Object {
    "description": undefined,
    "height": "400",
    "title": undefined,
    "url": "http://xhamster.com/xembed.php?video=7668106",
    "width": "510",
  },
}
`;
